nifi-docs/src/main/asciidoc/administration-guide.adoc

```diff
@@ -2083,11 +2187,11 @@ Once these State Providers have been configured in the _state-management.xml_ fi
 referenced by their identifiers.
 
 By default, the Local State Provider is configured to be a `WriteAheadLocalStateProvider` that persists the data to the
-_$NIFI_HOME/state/local_ directory. The default Cluster State Provider is configured to be a `ZooKeeperStateProvider`. The default
+`$NIFI_HOME/state/local` directory. The default Cluster State Provider is configured to be a `ZooKeeperStateProvider`. The default
 ZooKeeper-based provider must have its `Connect String` property populated before it can be used. It is also advisable, if multiple NiFi instances
 will use the same ZooKeeper instance, that the value of the `Root Node` property be changed. For instance, one might set the value to
 `/nifi/<team name>/production`. A `Connect String` takes the form of comma separated <host>:<port> tuples, such as
-my-zk-server1:2181,my-zk-server2:2181,my-zk-server3:2181. In the event a port is not specified for any of the hosts, the ZooKeeper default of
+my-zk-server1:2181,`my-zk-server2:2181,my-zk-server3:2181`. In the event a port is not specified for any of the hosts, the ZooKeeper default of
```

Comment:
I think the initial backtick should go before `my-zk-server1:2181`, so:

```
`my-zk-server1:2181,my-zk-server2:2181,my-zk-server3:2181`
```

Reply:
Good catch.